### ISSUE TITLE
Support orbital sets with energies

### ISSUE THREAD

**Issue by ghutchis** [OpenChemistry/chemicaljson#22]
The Avogadro implementation has "sets" as a nested object for orbital coefficients at each coordinate set (e.g., for following a reaction).

Right now, that does not include orbital energies which could also vary during a trajectory

Do we need to have varying orbital occupations?

**Comment by berquist** [comment]
If those orbitals could be natural, then yes.

**Comment by ghutchis** [comment]
Makes sense - I'll work on that. Formalizing the schema will be the first start.
